fluence-app-service/src/lib.rs

```diff
@@ -24,22 +24,17 @@
     unreachable_patterns
 )]
 
+mod config;
 mod errors;
 mod service;
+mod raw_toml_config;
 
 pub(crate) type Result<T> = std::result::Result<T, AppServiceError>;
 
 pub use errors::AppServiceError;
 pub use service::AppService;
 
-pub use fluence_faas::CallParameters;
-pub use fluence_faas::IValue;
-pub use fluence_faas::IType;
-pub use fluence_faas::FaaSInterface;
-pub use fluence_faas::RawModulesConfig;
-pub use fluence_faas::RawModuleConfig;
-pub use fluence_faas::ModulesConfig;
-pub use fluence_faas::ModuleConfig;
-pub use fluence_faas::WASIConfig;
-pub use fluence_faas::to_interface_value;
-pub use fluence_faas::from_interface_values;
+pub use config::*;
```

Comment:
I think it's better to be explicit about reimports. This way, I (as a developer) can take a look at lib.rs, and see right away what the library exposes to me. With `::*` approach, I have to dig inside each module, it's much-much harder to understand what's meant as API this way.

Reply:
It's from the experience, I just tried to look up what `RawModuleConfig` is now called, and I was surprised that it's not in lib.rs